distributed/utils.py

```diff
@@ -1605,15 +1603,7 @@ def clean_dashboard_address(addrs: AnyType, default_listen_ip: str = "") -> list
 
 
 _deprecations = {
-    "deserialize_for_cli": "dask.config.deserialize",
-    "serialize_for_cli": "dask.config.serialize",
-    "format_bytes": "dask.utils.format_bytes",
-    "format_time": "dask.utils.format_time",
-    "funcname": "dask.utils.funcname",
-    "parse_bytes": "dask.utils.parse_bytes",
-    "parse_timedelta": "dask.utils.parse_timedelta",
-    "typename": "dask.utils.typename",
-    "tmpfile": "dask.utils.tmpfile",
```

Comment:
All these were deprecated since 2021

Reply:
In the interest of a meaningful changelog, I'm inclined to split off these changes. The title of this PR will end up in the changelog and nothing in there says anything about removing deprecated functionality or configuration